src/main/java/gov/cabinetofice/gapuserservice/model/Blacklist.java

```diff
@@ -0,0 +1,24 @@
+package gov.cabinetofice.gapuserservice.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.*;
+import lombok.*;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.util.Date;
+
+@EntityListeners(AuditingEntityListener.class)
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Table
```

Comment:
Can we give this a more descriptive table name such as "token_blacklist" or "jwt_blacklist"